x-pack/plugins/security_solution/public/management/components/endpoint_response_actions_list/response_actions_log.test.tsx

```diff
@@ -409,6 +416,72 @@ describe('Response actions history', () => {
       );
     });
 
+    it('should contain download link in expanded row for `get-file` action WITH file operation permission', async () => {
+      const privileges = useUserPrivilegesMock();
+
+      useUserPrivilegesMock.mockImplementationOnce(() => {
+        return {
+          ...privileges,
+          endpointPrivileges: {
+            ...privileges.endpointPrivileges,
+            canWriteFileOperations: true,
+          },
+        };
+      });
+
+      mockUseGetEndpointActionList = {
+        ...baseMockedActionList,
+        data: await getActionListMock({ actionCount: 1, commands: ['get-file'] }),
+      };
+
+      render();
+      const { getAllByTestId } = renderResult;
+
+      const expandButtons = getAllByTestId(`${testPrefix}-expand-button`);
+      expandButtons.map((button) => userEvent.click(button));
+      const trays = getAllByTestId(`${testPrefix}-details-tray`);
+      expect(trays).toBeTruthy();
+      expect(
+        Array.from(trays[0].querySelectorAll('dd pre code'))
```

Comment:
could you use a `data-test-subj` instead of this css selector?  Doing so would be more stable case the component's internal implementation changes in the future. (same comment for other areas below)

Reply:
72aed7e3c97db58552342377db11fd361b4ffaa7

Reply:
72aed7e3c97db58552342377db11fd361b4ffaa7